src/database/scripts/dropDockerTestDatabase.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+# Copyright 2020 OCAD University
+#
+# Licensed under the New BSD license. You may not use this file except in
+# compliance with this License.
+
+# Shuts down the docker container running the database and removes the
+# containers.
+
+# Default values
+POSTGRES_MAIN_CONTAINER=${POSTGRES_MAIN_CONTAINER:-"postgresdb"}
+PGPORT=${PGPORT:-5432}
+PGUSER=${PGUSER:-"admin"}
+
+PGDATABASE="prefs_testdb"
```

Comment:
Looking at `package.json` where the `test` task is defined, as the `PGDATABASE` variable is configurable via an environment variable, it probably should be defined through `PGDATABASE=${PGDATABASE:-"prefs_testdb"}` instead of being hardcoded.

Reply:
Done.  Also, the issue with running `export PGDATABASE=prefs_testdb PGPORT=5433; npm run startTestServer` should be fixed.

> You probably are aware that tests are missing for some APIs such as `bulkQuery`, `loadFromJSON` etc.

The tests are/were there.  The test for `loadFromJSON()` was at [line 217](https://github.com/fluid-project/preferencesServer/blob/02fb74ca2fae176a2c96f187883fb581d485fe5c/src/database/src/test/testPostgresOperations.js#L217).  , The one for `bulkQuery()` was at [line 168](https://github.com/fluid-project/preferencesServer/blob/02fb74ca2fae176a2c96f187883fb581d485fe5c/src/database/src/test/testPostgresOperations.js#L168).  I've changed the test names to make it more obvious with the latest code. 

> Can you add a section in README about how to run tests and warn about running test will download a postgres docker image, as what you did in the comment?

Sure.